jobserver/migrations/0008_coredeveloper_to_staffareaadministrator.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 5.1.1 on 2024-10-02 11:00
+
+from django.db import migrations
+
+from jobserver.authorization.roles import StaffAreaAdministrator
+
+
+# We want to eventually remove the "real" CoreDeveloper so make a fake one here
+class CoreDeveloper:
```

Comment:
I think this idea works well, but I would like a slightly more in-depth explanation in comment.

Reply:
It turns out this isn't quite sufficient for this migration to run with the real `CoreDeveloper` deleted when there are `jobserver.authorization.roles.CoreDeveloper` role records in the db. If you can find a way to make is sufficient then that would make this PR much simpler and more robust!